lib/os.rb

```diff
@@ -0,0 +1,298 @@
+require 'rbconfig'

```

Comment:
moreover, please use this platform utility from vagrant instead: https://github.com/mitchellh/vagrant/blob/master/lib/vagrant/util/platform.rb